src/test/java/socialite/storage/ProfilePictureStorageManagerTest.java

```diff
@@ -0,0 +1,45 @@
+package socialite.storage;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+import socialite.testutil.Assert;
+
+public class ProfilePictureStorageManagerTest {
+
+    private static final String TEST_FILE_NAME = "TEST_FILE_NAME";
+    private static final Path FOLDER_PATH = ProfilePictureStorageManager.PROFILE_PIC_FOLDER_PATH;
+
+    private File invalidFile = Paths.get("some", "random", "nonexistent", "file").toFile();
+    private File validFile = Paths.get("src", "test", "data", "profilepictures", "simu.jpeg").toFile();
+    private ProfilePictureStorageManager manager = ProfilePictureStorageManager.getInstance();
+
+    @Test
+    public void saveInvalidFile_throwsIoexception() {
+        Assert.assertThrows(IOException.class, () -> manager.saveProfilePicture(invalidFile, "testName"));
+    }
+
+    @Test
+    public void saveNullFile_notSaved() throws Exception {
+        manager.saveProfilePicture(null, TEST_FILE_NAME);
```

Comment:
Since the methods can throw exceptions, maybe `assertDoesNotThrow` should be used. That way, I think we can also remove the `throws Exception` descriptor for the methods.

Reply:
Done!